Touch Bar Simulator/ToolbarSlider.swift

```diff
@@ -53,7 +53,16 @@ final class ToolbarSlider: NSSlider {
 final class MenubarSlider: NSSlider {
 	override init(frame: CGRect) {
 		super.init(frame: frame)
-		cell = ToolbarSliderCell(knob: makeKnob(fillColor: NSColor.controlTextColor.withAlphaComponent(1.0), borderColor: .systemGray))
+
+		let accentColor = NSColor.controlAccentColor.withAlphaComponent(1.0)
+		let dimmedAccentColor = accentColor.blended(withFraction: 0.35, of: .black) ?? accentColor
```

Comment:
I think this looks better:

```swift
let dimmedAccentColor = NSColor.controlTextColor
```

Reply:
I agree, and it looks okay in light mode now with the shadow.